index.html

```diff
@@ -4,37 +4,47 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Michael Luna's Personal Website</title>
-    <link href="/app/styles/tailwind_output.css" rel="stylesheet">
+    <title>Michael Luna, Software Engineer</title>
+    <link href="/app/styles/tailwind_output.css?v=1.0.2" rel="stylesheet">
```

Comment:
Because the browser kept caching the CSS file, i decided to add version to the file